src/main/java/filters/ConflictFilter.java

```diff
@@ -0,0 +1,44 @@
+package filters;
+
+import entities.Schedule;
+import entities.Section;
+import entities.Timeslot;
+import java.util.ArrayList;
+import java.util.Collections;
+
+public class ConflictFilter implements Filter {
+    private boolean allowConflicts;
+
+    public ConflictFilter(boolean allowConflicts) {
```

Comment:
Does it make sense for this filter to be able to choose to allow conflicts? Seems kinda redundant.

Reply:
u right